### ISSUE TITLE
Module doesn't work with txtai 7. 

### ISSUE THREAD

**Issue by dustyatx** [qdrant/qdrant-txtai#7]
Tried to install and it broken my python environment for txtai.. I realized after that this is pinned to v5. I tried bumping up pyproject.toml got it to install but it didn't write anything to Qdrant.. 

Any chance you'd revisit this and update it?

**Comment by kacperlukawski** [comment]
Hello @dustyatx We definitely need to have a closer look, but this integration was not actively maintained for a while. We are open to contributions if you were successful in using it with the most recent version of txtai.

**Comment by miary** [comment]
Confirming that qdrant-txtai does not work with version 7. Really too bad, it would have been so powerful to combine both!!

**Comment by kacperlukawski** [comment]
@dustyatx @miary We've just released 2.0.0. Could you please upgrade?

**Comment by miary** [comment]
Perfect!!! Excellent upgrade. I tested qdrant-txtai 2.0.0, qdrant , run qdrant latest through Docker, txtai 7.1.0. 
The out-of-the-box test script works really great: 
import txtai
import os

os.environ["config"] = "app.yml"

embeddings = txtai.Embeddings()
embeddings.index(["Correct", "Not what we hoped"])
result = embeddings.search("positive", 1)
print(result)


However, I could not find any way to query an existing collection. I tried this as app.yml but it didn't work:
embeddings:
  path: sentence-transformers/all-MiniLM-L6-v2
  backend: qdrant_txtai.ann.qdrant.Qdrant
  metric: cosine
  qdrant:
    host: localhost
    port: 6333
    grpc_port: 6334
    prefer_grpc: true
    collection: startups
    search_params:
          hnsw_ef: 256

What is the way to query from an existing collection?
